lib/ansible/modules/yum_repository.py

```diff
@@ -50,6 +50,12 @@
       - Relative cost of accessing this repository. Useful for weighing one
         repo's packages as greater/less than any other.
     type: str
+  countme:
+    description:
+      - Whether a special flag should be added to a randomly chosen query each week.
```

Comment:
Should we be more specific? Copy&pasting from the man page:
```suggestion
      - Whether a special flag should be added to a randomly chosen metalink/mirrorlist query each week.
        This allows the repository owner to estimate the number of systems consuming it. 
```

Reply:
Absolutely, I'll update later